### PR TITLE
Added a new route /whitepaper and refactored the Page component - Issue 380 

### PR DESCRIPTION
**Pull request by Phasor**
Changes made in relation to Github airswap-web issue 380:

1. Refactored the Page Component
- not tightly coupled with the SwapWidget component anymore
- Page component is now a generic page template, that we can inject child components into (like Whitepaper)

2. Added /whitepaper route
- added Whitepaper component and added route to App.tsx

### CODE REVIEW

**Comment by dmosites** [comment]
Thanks @Phasor! The whitepaper is currently just HTML / CSS with some images as attached. Also fine if the URL ends up being `airswap.io/whitepaper.htm` if it doesn't fit into the current structure.
[whitepaper.zip](https://github.com/airswap/airswap-web/files/7940624/whitepaper.zip)

**Review by robbestad — CHANGES_REQUESTED** [review]

File: src/components/Whitepaper/Whitepaper.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  return (
+    <>
+      <Page>
+        <div>
+          <a
+            href="https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf"
+            download
+            target="_blank"
+          >
+            Download Airswap Whitepaper.
```
Comment (piersss) [inline src/components/Whitepaper/Whitepaper.tsx:15]: A translation should be added for this.

**Review by piersss — CHANGES_REQUESTED** [review]
Hello Phasor, welcome to the team!

I think the HomePage and Whitepaper should go in their own src/pages folder. Also try to use styled components and not place div's and a's in your components.

**Comment by Phasor** [comment]
Having issues with this branch. Closing and will open a new Issue 380 PR from a new branch.

**Pull request closed without merging**